acceptor/tcp_acceptor.go

```diff
@@ -78,21 +77,26 @@ func (t *tcpPlayerConn) GetNextMessage() (b []byte, err error) {
 
 // NewTCPAcceptor creates a new instance of tcp acceptor
 func NewTCPAcceptor(addr string, certs ...string) *TCPAcceptor {
-	keyFile := ""
-	certFile := ""
+	certificates := []tls.Certificate{}
 	if len(certs) != 2 && len(certs) != 0 {
 		panic(constants.ErrInvalidCertificates)
 	} else if len(certs) == 2 {
-		certFile = certs[0]
-		keyFile = certs[1]
+		cert, err := tls.LoadX509KeyPair(certs[0], certs[1])
+		if err != nil {
+			panic(constants.ErrInvalidCertificates)
```

Comment:
It might be good to add a test validating this behavior.

Reply:
There are some already: https://github.com/topfreegames/pitaya/pull/284/files#diff-91d8960c4bbf5e4ec59f1cc6232c15ea612244fef31e5a2e8c49edbd4c348997L52

Reply:
These current tests only seem to assert the case when it panics when `len(certs)` is 0 or different than 2. I believe we need to add a new entry to the test table with two invalid entries in `certs` so it tries to load the certificate and then panics.